calliope/backend/pyomo/util.py

```diff
@@ -199,3 +201,22 @@ def loc_tech_is_in(backend_model, loc_tech, model_set):
         return True
     else:
         return False
+
+
+def get_domain(var: xr.DataArray) -> str:
+    def check_sign(var):
+        if "cost" in var.name or var.name in ["resource", "loc_coordinates"]:
```

Comment:
I guess there are several kinds of `cost`s?  This is not very readable, I would simply use `re.match`.
```suggestion
        if  re.match("resource|loc_coordinates|cost*", var.name):
```
Don't forget to `import re`.

Reply:
Yeah, I always avoided re so that I didn't have to import it; no longer sure why I had that mentality, it certainly makes the expression more readable